Sample Data/Custom/DSTIMAccess_CL.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Should be an array, braces [ and ] are missing.

Reply:
@v-jayakal fixed